notification/src/test/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+#spring.jpa.show-sql=true
+#spring.jpa.properties.hibernate.format_sql=true
+
+#logging.level.org.hibernate.SQL=DEBUG
```

Comment:
It is not a good idea to set default log level to DEBUG or TRACE. In dev profile it is OK.